database/migrations/2022_10_19_014608_create_game_user_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('game_user', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('game_id');
```

Comment:
Should there be a ->constrained(); attached to game_id here? Is it referencing game_id in category_game?

![Screen Shot 2022-10-19 at 11 43 32 AM](https://user-images.githubusercontent.com/95186966/196740126-7fe0011b-ea50-4424-941e-1e829682fab7.png)

Reply:
@lrapp1 is correct, we do need the `->constrained()` method to create the foreign key constraint. that or the more verbose `->references(column)->on(table)` but we should be able to use the Laravel naming conventions.

Reply:
Added!